include/view_templates.h

```diff
@@ -164,5 +165,6 @@
 #define UI_BACKGROUND_LEFT_RIGHT_ICONS \
```

Comment:
Should this have a different name?
This change will affect all other apps and all screenshots.

Reply:
Actually we are not using that. It's just a define for old UI in NanoX/SP. I think that we can remove it safely 🙂